test/unit/specs/components/governance/PageGovernance.spec.js

```diff
@@ -43,8 +44,8 @@ describe(`PageGovernance`, () => {
   })
 
   it(`should show the search on click`, () => {
-    wrapper.find(`.tm-tool-bar i.search`).trigger(`click`)
-    expect(wrapper.contains(`.tm-modal-search`)).toBe(true)
+    wrapper.vm.$el.querySelector(`.tm-tool-bar a.search-button i`).click()
```

Comment:
find didn't work ?

Reply:
I'll try again with find + trigger.
The `trigger` vs `click` convinced me to switch when I was doing this for some reason trigger wasn't working

Reply:
Yep it's working, Thanks! I'll revert to trigger('click')